### DISCUSSION TITLE
Traffic intersections we can target

### DISCUSSION THREAD

**Discussion by ahmedsadatsakif** [Mechyard/traffic-concepts-and-ideas#5] [Ideas]
Everyone please suggest the traffic intersections we should target initially and provide an idea of why you think this intersection might be a good idea.

**Comment by 01646489831** [comment]
Abdullahpur, Housebuilding area. Uttara

**Reply by csilm** [reply]
Gulshan
Banani

**Reply by faisal07bd** [reply]
Gulshan and banani and mauy bijay sharani and mirpur 10 Gol chokkor.

**Comment by ahmedsadatsakif** [comment]
So far, we got suggestion about Gulistan, Gulshan, Mohakhali, Bijoy Sharani, Dhanmondi 27, Dhanmondi Mirpur Road near Science Lab, Mirpur 10, Uttara, House building and Abdullahpur.

**Comment by csilm** [comment]
Mirpur 10

**Comment by ahmedsadatsakif** [comment]
As per our previous discussion, Gulshan and Bijoy Sharani was a good idea since it opens up opportunity to use the existing infrastructure like cameras, road signals and any other utilities not being used effectively. 

Also it will help to bring us media attention to reach out to a wider community.

**Comment by ahmedsadatsakif** [comment]
![Screenshot_20240821-180059](https://github.com/user-attachments/assets/e40026cc-e2f6-4286-974c-afdaee71aee7)
How about Gulshan 1 circle? There is always extreme traffic during office hours and I think it will provide a large list of interested people who will volunteer for a survey.

**Reply by faisal07bd** [reply]
yes gulshan 1 and 2 intersection can give us a lots of DATA

**Comment by samiulhasanocean** [comment]
Kazla, Soniakhra  flyover up side ........ and same gulistan flyover down side.[https://maps.app.goo.gl/yVjYseGwm6gxz1Bc6](url)  and[https://maps.app.goo.gl/QjuCLEG8BpxqaGaJ6](url)   
![gulistan](https://github.com/user-attachments/assets/85cc1e05-0ef8-4c40-9d3a-fe2ffd3c2498)

**Comment by ahmedsadatsakif** [comment]
Would it be possible for some of us to observe and share their findings about these intersections during different times of the day? Like how the traffic becomes congested and how does it loosen up. 

We might be able to come up with a pattern based on the data.

**Reply by 01646489831** [reply]
For that we should take help from students volunteers beacuase we may not give the time in road as we should attend office.

**Reply by faisal07bd** [reply]
our initial plan was to engage ourselves to collect the data.As engaging anyone would cost us money.but your idea is great.

**Reply by 01646489831** [reply]
Yeah we should! 
Though It's difficult, but we should think in a different way

**Comment by csilm** [comment]
https://docs.google.com/forms/d/e/1FAIpQLSfBeG3shXX_MxH2lXaKFh9nqekRdlCxwekyOSUgf00CPv4c6Q/viewform?pli=1&pli=1

I have created this form to collect data. Please let me know if you have any suggestions to add any other questions. It can also be in Bengali. If you guys say I will convert it to Bengali.

**Reply by faisal07bd** [reply]
I already shared this doc link in a facebook group about traffic. should I delete the post or keep it continiue

**Reply by csilm** [reply]
Yeah. Delete this. Some information needs to change. I/ the team will give a msg when we start collecting the data on this form.

**Reply by faisal07bd** [reply]
ok copied. was busy with flood effected people fund collection
